uber/models.py

```diff
@@ -1276,12 +1276,13 @@ def merch(self):
     @property
     def accoutrements(self):
         stuff = [] if self.ribbon == c.NO_RIBBON else ['a ' + self.ribbon_label + ' ribbon']
-        stuff.append('a {} wristband'.format(c.WRISTBAND_COLORS[self.age_group]))
+        if c.WRISTBANDS_ENABLED:
+            stuff.append('a {} wristband'.format(c.WRISTBAND_COLORS[self.age_group]))
         if self.regdesk_info:
             stuff.append(self.regdesk_info)
         if self.amount_extra >= c.SUPPORTER_LEVEL:
             stuff.append('their Supporter badge')
-        return comma_and(stuff)
+        return (' with ' if stuff else '') + comma_and(stuff)
```

Comment:
Most people now have literally nothing other than their badge which they receive at regdesk.  We therefore don't want to say "with" followed by nothing.  So instead we now only put the word "with" in the message if there's at least one thing they're being given.